### PR TITLE
fix: added app_namespace and on_delete that was needed for django (juniper migration)

### PR DESCRIPTION
**Pull request by MaferMazu**
- I added the app_namespace that was needed [(doc)](https://docs.djangoproject.com/en/3.2/ref/urls/#include)
- Now the on_delete is needed in OneToOne fields, and it is cascade because it was what was by default before. [(doc)](https://docs.djangoproject.com/en/3.2/ref/models/fields/#arguments)

### CODE REVIEW

File: campusromero_openedx_extensions/general_custom_views/api/urls.py [hunk 1]
```diff
@@ -7,8 +7,9 @@
     url(
         r'^v1/',
         include(
-            'campusromero_openedx_extensions.general_custom_views.api.v1.urls',
-            namespace='camrom-api'
+            ('campusromero_openedx_extensions.general_custom_views.api.v1.urls',
+            'camrom-api'),
+            namespace='camrom-api',
```
Comment (mariajgrimaldi) [inline campusromero_openedx_extensions/general_custom_views/api/urls.py:12]: Why is this needed? 
` include((pattern_list, app_namespace), namespace=None)`
That format receives a pattern list, not the module (correct me if I'm wrong)

Comment (MaferMazu) [reply campusromero_openedx_extensions/general_custom_views/api/urls.py:12]: To be honest, initially I put it like this because I saw it in an example but I was trying to place it in different ways and in all of them I get an error.

Like it was before:
```python
url(
        r'^v1/',
        include(
            'campusromero_openedx_extensions.general_custom_views.api.v1.urls',
            namespace='camrom-api'
        )
```
Screenshot from 2021-09-17 10-42-46

Other tests:

```python
url(
        r'^v1/',
        include(
            'campusromero_openedx_extensions.general_custom_views.api.v1.urls',
            app_name='camrom-api',
            namespace='camrom-api'
        )
```
Screenshot from 2021-09-17 10-44-45


```python
url(
        r'^v1/',
        include(
            module=campusromero_openedx_extensions.general_custom_views.api.v1.urls,
            namespace='camrom-api'
        )
```
Screenshot from 2021-09-17 10-47-44

```python
url(
        r'^v1/',
        include(
            module='campusromero_openedx_extensions.general_custom_views.api.v1.urls',
            namespace='camrom-api'
        )
```
Screenshot from 2021-09-17 10-48-36

✅ The one that worked is this option:
```python
url(
        r'^v1/',
        include(
            ('campusromero_openedx_extensions.general_custom_views.api.v1.urls',
            'camrom-api'),
            namespace='camrom-api',
        )
```

**Review by MoisesGSalas — APPROVED** [review]

File: campusromero_openedx_extensions/general_custom_views/api/urls.py (hunk 1 above)
Comment (MoisesGSalas) [reply campusromero_openedx_extensions/general_custom_views/api/urls.py:12]: A module is a list of patterns, as long as it defines the `urlpatterns` list. See https://github.com/django/django/blob/2da029d8540ab0b2e9edcba25c4d46c52853197f/django/urls/conf.py#L35.

If you don't define namespace [it uses app_name](https://github.com/django/django/blob/2da029d8540ab0b2e9edcba25c4d46c52853197f/django/urls/conf.py#L44).

**Review by mariajgrimaldi — APPROVED** [review]
